pkgs/development/python-modules/hdmedians/default.nix

```diff
@@ -0,0 +1,34 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, nose
+, cython
+, numpy
+}:
+
+buildPythonPackage rec {
+  version = "0.13";
+  pname = "hdmedians";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "230f80e064d905c49a1941af1b7e806e2f22b3c9a90ad5c21fd17d72636ea277";
+  };
+
+  buildInputs = [ nose cython ];
```

Comment:
nose is a test runner and should go into `checkInputs`

Reply:
fixed.